### PR TITLE
June OOB KDM release 

### PR DESCRIPTION
**Pull request by kinarashah**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1129,6 +1129,22 @@ releases:
         repo: rancher-rke2-charts
         version: 2.11.100-build2023051508
     featureVersions: *featureVersions-v1
+  - version: v1.24.15+rke2r1
+    minChannelServerVersion: v2.7.5-alpha1
```
Comment (rancher-max) [inline channels-rke2.yaml]: This should be `v2.6.7-alpha1` -- we got it wrong in our original PR I guess

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1267,13 +1283,29 @@ releases:
         repo: rancher-rke2-charts
         version: 1.7.101
     featureVersions: *featureVersions-v1
+  - version: v1.25.11+rke2r1
+    minChannelServerVersion: v2.7.5-alpha1
```
Comment (rancher-max) [inline channels-rke2.yaml]: This should be `v2.7.2-alpha1` -- we got it wrong in our original PR I guess

**Review by rancher-max — COMMENTED** [review]
There were two mistakes made by our team in the rke2 PR. We can fix and make a new PR to this branch, or we can fix as part of this one. Whichever is fine by me

**Review by snasovich — DISMISSED** [review]

**Comment by snasovich** [comment]
> There were two mistakes made by our team in the rke2 PR. We can fix and make a new PR to this branch, or we can fix as part of this one. Whichever is fine by me

@rancher-max , good catch. I recommend having it fixed in a separate PR against `dev-v2.7-2023-06-patches` branch so that branch is correct as well.

**Review by snasovich — DISMISSED** [review]

**Comment by jiaqiluo** [comment]
Hi @HarrisonWAffel,  because the base branch does not contain the updates on the drone CI in the `dev-v2.7` branch, the automated image-mirroring and image-checking will not happen when this PR is merged. 

Let's cherry-pick those from the dev-v2.7 branch into the base branch. WDYT?

**Comment by HarrisonWAffel** [comment]
@jiaqiluo sounds good to me, are you able to cherry pick them or should I? Is there a single commit which introduced the auto image-mirroring, or multiple?

**Comment by jiaqiluo** [comment]
@HarrisonWAffel, there are multiple commits.  I am going to raise a PR to the base branch in a bit.

**Review by rancher-max — DISMISSED** [review]
LGTM for k3s/rke2

**Review by rancher-max — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**